src/app/modules/sgid/__tests__/sgid.service.spec.ts

```diff
@@ -68,14 +73,36 @@ describe('sgid.service', () => {
       const url = SgidService.createRedirectUrl(
         MOCK_DESTINATION,
         MOCK_REMEMBER_ME,
+        SGID_DEFAULT_ATTR_LIST,
       )
       expect(url._unsafeUnwrap()).toEqual(MOCK_REDIRECT_URL)
       expect(sgidClient.authorizationUrl).toHaveBeenCalledWith(
         MOCK_STATE,
-        SGID_SCOPES,
+        SGID_SINGPASS_SCOPE,
         null,
       )
     })
+
+    it('should return extract additional OAuth scopes from MyInfo', () => {
```

Comment:
The English sounds funny? 😅 Did you mean:

```suggestion
    it('should extract additional OAuth scopes from MyInfo', () => {
```